README.md

```diff
@@ -28,7 +28,7 @@ At the moment Theia is still in [active development](https://github.com/theia-id
 
 This script pulls the image and runs Theia IDE on http://localhost:3000 with the current directory as a workspace. The option of `--init` is added to fix the [defunct process problem](https://github.com/theia-ide/theia-apps/issues/195).
 
-    # Linux or macOS
+    # Linux, macOS, or PowerShell
```

Comment:
I'm not sure how we should organize the following information, but I think we should keep the categories based on operating-system and not display PowerShell at this level to avoid confusion.

I think we should create a single pull-request with the following changes and those present in [#388](https://github.com/theia-ide/theia-apps/pull/338/files), keep the original categories, and under Windows we display the two options, one for cmd and one for powershell.

Reply:
PowerShell can be installed on Ubuntu as well... But it is more commonly used on Windows indeed.

Reply:
I think it is fine as it is right now, at least it doesn't bother me.

Reply:
I mentioned PowerShell separately for the reason @marechal-p mentioned - the open source version can be installed on other operating systems as well. If you ask me personally, the line referring to CMD could rather be dropped because Microsoft pushes people to use PowerShell more and more.